Sources/VLCMedia.m

```diff
@@ -188,10 +188,7 @@ + (NSString *)codecNameForFourCC:(uint32_t)fourcc trackType:(NSString *)trackTyp
         track_type = libvlc_track_text;
 
     const char *ret = libvlc_media_get_codec_description(track_type, fourcc);
-    if (ret)
-        return [NSString stringWithUTF8String:ret];
-
-    return @"";
+    return [NSString stringWithUTF8String:ret];
```

Comment:
this will crash with ret = nil. This was not redundant line

Reply:
I tracked it down and seems like `*ret = libvlc_media_get_codec_description(...)` calls `vlc_fourcc_GetDescription`, which has the following return statement

```c
return LookupCat(fourcc, &ret, cat) ? ret : "";
```

Doesn't this mean that `vlc_fourcc_GetDescription` will never return nil?